backend/src/controllers/organizationController.js

```
@@ -27,28 +27,37 @@ const updateApplicationStatus = async (req, res) => {
   }
 };
 
+
 const getOrganizationApplications = async (req, res) => {
   try {
     const organizationId = req.user.organizationId;
-    const result = await organizationService.getOrganizationApplications(organizationId);
+    const { page = 1, limit = 10 } = req.query;  
+    const offset = (parseInt(page) - 1) * parseInt(limit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)